voltcraft/pps.py

```diff
@@ -132,8 +157,8 @@ def output(self, state: int) -> None:
         self._query("SOUT%d" % state)
 
     def voltage(self, voltage: float) -> None:
-        """set voltage: silently saturates at 0 and VMAX"""
-        voltage = min(max(0, int(voltage * 10)), int(self._vmax * 10))
+        """set voltage: silently saturates at VMIN and VMAX"""
+        voltage = min(max(self._vmin * 10, int(voltage * 10)), int(self._vmax * 10))
```

Comment:
We should cast this to `int` to be clear.

```suggestion
        voltage = min(max(int(self._vmin * 10), int(voltage * 10)), int(self._vmax * 10))
```

Reply:
Thank you for the suggestion, oversight on my ended. It is now corrected.